cloudslang-compiler/src/test/java/io/cloudslang/lang/compiler/modeller/ExecutableBuilderTest.java

```diff
@@ -312,43 +312,6 @@ public void operationWithEmptyActionDataThrowException() throws Exception {
         transformToExecutable(mockParsedSlang, operationName, executableRawData);
     }
 
```

Comment:
because of the restructuring, now the logic of key validation is moved to the java/python transformers

tests are added there: e.g. https://github.com/CloudSlang/cloud-slang/pull/677/files#diff-6e4240d6335bba4b8445785e3e39aa6dR87